truncate_client/src/regions/active_game.rs

```diff
@@ -198,8 +200,24 @@ impl ActiveGame {
                             ui.add_space(10.0);
                         }
 
-                        ui.add_space(10.0);
+                        ui.add_space(5.0);
                     }
+
+                    if self.ctx.is_mobile {
+                        let text = TextHelper::heavy("VIEW INFO", 12.0, ui);
```

Comment:
Stoked to have this feature 🎉 
Mobile playable now??

Reply:
Tis! Probably!